templates/base.html

```diff
@@ -1,73 +1,48 @@
 {% load static %}
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
What did you change in the base file? Can you fix it so it will be easier to see the difference?

Reply:
no code was supposed to change here 
you reformatted code that you didn't change by mistake please change it back